declarations_site/chatbot/utils.py

```diff
@@ -0,0 +1,89 @@
+import requests
+from datetime import datetime
+from django.conf import settings
+from django.core.cache import cache
+from elasticsearch_dsl import Search
+from catalog.constants import CATALOG_INDICES
+from catalog.utils import base_search_query
+from chatbot.models import ChatHistory
+
+
+def ukr_plural(value, *args):
+    value = int(value)
+    rem = value % 10
+    if value > 4 and value < 20:
+        return args[2]
+    elif rem == 1:
+        return args[0]
+    elif rem > 1 and rem < 5:
+        return args[1]
+    else:
+        return args[2]
+
+
+def simple_search(query, deepsearch=False):
+    base_search = Search(index=CATALOG_INDICES)
+    sort = 'general.full_name_for_sorting'
+    source = ['meta.id', 'general.*', 'intro.*', 'declaration.*']
+
+    search = base_search_query(base_search, query, deepsearch, {})
+    search = search.sort(sort).source(source)
+
+    if not hasattr(search, 'found_total'):
+        search.found_total = search.count()
+
+    return search
+
+
+def client_credentials():
+    cached_creds = cache.get(settings.BOTAPI_APP_ID)
```

Comment:
Add those keys to settings.py with empty values.

Otherwise by default it'll throw an error.